README.md

```diff
@@ -38,38 +39,37 @@ The example contains an [mbed_app.json](./mbed_app.json) configuration file that
 ## Hardware requirements
 
 KVStore general API may use several types of key/value storage classes, according to the configuration,  that run on top of an underlying Block Device. Each type has different HW requirements for the underlying Block Device.
+
 1. TDB_INTERNAL - may run on top of flash Block Device only (typically internal flash)
-2. TDB_EXTERNAL - requires external flash for the key/value storage and typically internal flash for its replay protection storage.
-3. TDB_EXTERNAL_NO_RBP - requires external flash for the key/value storage
-4. FILESYSTEM - may use either flash or SD card for the key/value storage and typically internal flash for its replay protection storage.
-5. FILESYSTEM_NO_RBP - may use either flash or SD card for the key/value storage
+1. TDB_EXTERNAL - requires external flash for the key/value storage and internal flash for its replay protection storage.
+1. FILESYSTEM - may use either flash or SD card for the key/value storage and internal flash for its replay protection storage.
 
-## License and contributions 
+## License and contributions
 
-The software is provided under [Apache-2.0 license](./LICENSE). Contributions to this project are accepted under the same license. Please see [contributing.md](./CONTRIBUTING.md) for more info. 
+The software is provided under [Apache-2.0 license](./LICENSE). Contributions to this project are accepted under the same license. Please see [contributing.md](./CONTRIBUTING.md) for more info.
 
 This project contains code from other projects. The original license text is included in those source files. They must comply with our [license guide](https://os.mbed.com/docs/latest/reference/license.html)
 
 ## Usage
 
-#### Import the example
+### Import the example
 
 Make sure you have an Mbed development environment set up. [Get started with Mbed OS](https://os.mbed.com/docs/latest/tutorials/your-first-program.html)
 to set everything up.
 
 From the command-line, import the example:
 
-```
+``` commandline
```

Comment:
Not sure this is necessary. It won't make any difference, it's a supported language, see https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5285-L5289
We could either leave it out as we do else where, or use the closest (e.g. `sh`).